app/src/test/java/com/client/shop/ui/account/PersonalInfoActivityTest.kt

```diff
@@ -0,0 +1,122 @@
+package com.client.shop.ui.account
+
+import android.content.Context
+import android.view.inputmethod.EditorInfo
+import com.client.shop.TestShopApplication
+import com.client.shop.gateway.entity.Customer
+import com.nhaarman.mockito_kotlin.*
+import kotlinx.android.synthetic.main.activity_personal_info.*
+import org.junit.After
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE, application = TestShopApplication::class)
+class PersonalInfoActivityTest {
+
+    private lateinit var activity: PersonalInfoActivity
+    private lateinit var context: Context
+
+    private val oldCustomer: Customer = mock {
+        on { email } doReturn "mail@mail.com"
+        on { firstName } doReturn "firstName"
+        on { lastName } doReturn "lastName"
+        on { phone } doReturn "06332916"
+    }
+
+    @Before
+    fun setUpTest() {
+        activity = Robolectric.setupActivity(PersonalInfoActivity::class.java)
+        context = RuntimeEnvironment.application.baseContext
+        doNothing().`when`(activity.presenter).getCustomer()
+    }
+
+    @After
+    fun tearDown() {
+        activity.finish()
+    }
+
+    @Test
+    fun shouldShouldLoadCurrentCustomerOnLoadData() {
+        activity.loadData()
+        verify(activity.presenter, times(2)).getCustomer()
```

Comment:
Why this method needs to be called twice?

Reply:
this method called from on create in activity
when robolectrick create activity it called first time
then when we test load data method we call it manualy, and counter increments one more time

Reply:
It's ok